source/opt/pass.cpp

```diff
@@ -106,6 +102,17 @@ bool Pass::ProcessCallTreeFromRoots(
   return modified;
 }
 
+Pass::Status Pass::Run(ir::IRContext* ctx) {
+  Pass::Status status = Process(ctx);
```

Comment:
Note for subsequent PRs.  This is where we'd move the call to Pass::InitializeProcessing(), right?

Reply:
Yes it is.